### PR TITLE
[MIG] partner_contact_job_position: Migration to 10.0

### PR DESCRIPTION
**Pull request by chienandalu**
- Minor migration tweak

cc @Tecnativa

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional and code review

File: partner_contact_job_position/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+from odoo import models, fields
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    job_position_id = fields.Many2one(
+        "res.partner.job_position",
+        "Categorized job position",
+        oldname="job_position")
```
Comment (yajo) [inline partner_contact_job_position/models/res_partner.py]: If oldname was already there in 9.0, no need to keep it for 10.

**Review by yajo — APPROVED** [review]
Code OK

**Comment by yajo** [comment]
Travis is ❌

**Comment by rafaelbn** [comment]
@Yajo travis nor related

**Review by rafaelbn — APPROVED** [review]
Tested in runbot 👍

**Comment by yajo** [comment]
Travis fails, I feel it's a problem in the main branch.

**Comment by rafaelbn** [comment]
Travis not related as said

**Pull request merged**